src/Explorer/Explorer.ts

```diff
@@ -424,8 +424,8 @@ export default class Explorer {
     this.shouldShowShareDialogContents = ko.observable<boolean>(false);
     this.shouldShowDataAccessExpiryDialog = ko.observable<boolean>(false);
     this.shouldShowContextSwitchPrompt = ko.observable<boolean>(false);
-    this.isGalleryPublishEnabled = ko.computed<boolean>(() =>
-      this.isFeatureEnabled(Constants.Features.enableGalleryPublish)
+    this.isGalleryPublishEnabled = ko.computed<boolean>(
+      () => this.isFeatureEnabled(Constants.Features.enableGalleryPublish) || configContext.ENABLE_GALLERY_PUBLISH
```

Comment:
>ENABLE_GALLERY_PUBLISH [](start = 92, length = 22)

nit: move this first. 
feature enabled is probably an iterator and you can avoid that if the direct read of config was true.